arraycontext/container/traversal.py

```diff
@@ -570,14 +573,35 @@ def thaw(ary: ArrayOrContainerT, actx: ArrayContext) -> ArrayOrContainerT:
         in :mod:`meshmode`. This was necessary because
         :func:`~functools.singledispatch` only dispatches on the first argument.
     """
+    if __debug__:
```

Comment:
Likewise for `thaw`: I think that no longer needs `@singledispatch`.

Reply:
Yep, removed!